File: js/ejer24.js
//solicitar al usuario un año
año = 2020 //año a evaluar

//evaluar si un año es bisiesto o no

if ((año % 4 === 0 && año % 100 !== 0) || (año % 400 === 0)) {
        console.log(año + " es un año bisiesto.");
    } else {
        console.log(año + " no es un año bisiesto.");
    }

File: python/ejer24.py
#solicitar año e indicar si es bisiesto
año = 2020    # Año a evaluar

#evaluar si el año es bisiesto o no
if (año % 4 == 0 and año % 100 != 0) or (año % 400 == 0):
    print(str(año) + " es un año bisiesto.")
else:    
    print(str(año) + " no es un año bisiesto.")
